apps/server/src/adapters/OscAdapter.ts

```diff
@@ -37,7 +37,14 @@ export class OscServer implements IAdapter {
       }
 
       try {
-        const reply = dispatchFromAdapter(path, args, 'osc');
+        const reply = dispatchFromAdapter(
+          path,
+          {
+            payload: args,
+            params,
+          },
+          'osc',
+        );
         if (reply) {
           const { topic, payload } = reply;
           this.osc.emit(topic, payload);
```

Comment:
have you tested this to work?

Reply:
it works in general, I didn't know we needed to reply to `change`.

Would `{ topic: propertyName, payload: newValue };` be a proper reply then?

Reply:
sorry this was my mistake, we don't expect a reply from change